lib/logstash/bundler.rb

```diff
@@ -0,0 +1,125 @@
+require "bundler"
```

Comment:
well, its not only bundler patches, its also the invokation code. I decided to put anything directly bundler specific into `LogStash::Bundler`. we can split but really, I don't think its necessary.